api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -151,6 +151,12 @@ private void sendAlertsForCostThresholds(
         });
   }
 
+  // are these two cost values approximately equal?
+  // used to determine whether we should log anomalies
+  private boolean withinCostTolerance(double currentCost, double previousCost) {
```

Comment:
Please check if there's a library function for this. As excited as I am to see actual math for the first time in 2020, we shouldn't be doing it this way. If we keep this, we probably need to make a constant out of the tolerance.

Additionally, why are we using doubles instead of a Currency class?

Reply:
good call re: library function.  I'll update.

re: Currency.  When implemented, that seemed like overkill for our application since we're not doing anything complex and we don't require financial-level precision.  I'm open to the change if we think it's justified.

Reply:
Ultimately we retrieve these values from BigQuery as doubles, so we'll never have the integer-like precision we'd really want in a Currency.